tasklib/tests.py

```diff
@@ -839,16 +843,17 @@ class TaskFromHookTest(TasklibTest):
         '"status":"pending",'
         '"uuid":"81305335-0237-49ff-8e87-b3cdc2369cec"}')
 
-    input_modify_data = six.StringIO(input_add_data.getvalue() + '\n' +
+    input_modify_data = six.StringIO(
+        input_add_data.getvalue() + '\n' +
         '{"description":"Buy some milk finally",'
         '"entry":"20141118T050231Z",'
         '"status":"completed",'
         '"uuid":"a360fc44-315c-4366-b70c-ea7e7520b749"}')
 
     exported_raw_data = (
         '{"project":"Home",'
-         '"due":"20150101T232323Z",'
-         '"description":"test task"}')
+        '"due":"20150101T232323Z",'
+        '"description":"test task"}')
```

Comment:
The closing `)` should be on a new line here, since the opening one is on its own line.

Reply:
Fixed in 8f4fabf